packages/http/fetch/src/fetchRequestAdapter.ts

```diff
@@ -356,8 +359,8 @@ export class FetchRequestAdapter implements RequestAdapter {
 		});
 	};
 	public enableBackingStore = (backingStoreFactory?: BackingStoreFactory): void => {
-		this.parseNodeFactory = enableBackingStoreForParseNodeFactory(this.parseNodeFactory);
-		this.serializationWriterFactory = enableBackingStoreForSerializationWriterFactory(this.serializationWriterFactory);
+		this.parseNodeFactory = enableBackingStoreForParseNodeFactory(this.parseNodeFactory as ParseNodeFactoryRegistry, this.parseNodeFactory);
```

Comment:
we should type test instead of using as here.

Reply:
@baywet, I think we can also move this functions to instance methods as well? I believe all the functions in file `apiClientBuilder.ts` can be moved to instance methods in the registries

Reply:
The error should be resolved, I'm currently type testing